src/sdk/pynni/nni/compression/torch/pruners.py

```diff
@@ -165,9 +170,8 @@ def update_epoch(self, epoch):
 
         if epoch > 0:
             self.now_epoch = epoch
-            for k in self.if_init_list.keys():
-                self.if_init_list[k] = True
-
+            for wrapper in self.get_modules_wrapper():
+                wrapper.registered_buffers['if_calculated'].copy_(torch.tensor(0)) # pylint: disable=not-callable
```

Comment:
which one is better? `wrapper.registered_buffers['if_calculated']` or `wrapper.if_calculated`

Reply:
Just tested, `wrapper.if_calculated` does not work,  there is no error reported, but the value is still `1` we use  `wrapper.if_calculated.copy_(torch.tensor(0))`